### PR TITLE
Anh Dang

### PR DESCRIPTION
**Pull request by anhcdang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,19 +45,21 @@ Open the `index.html` file in your browser and check the console in Chrome Dev T
...
 
 Now that you have access to the `h1#header` element, use Javascript to change the element's font color to your favorite color.
 
-**YOUR NOTES**
-```
-
+**YOUR NOTES:**
+To change font color, first access the `style` property, then change font color:
+```js
+header.style.color = '#088F8F'
```
Comment (BrendaMichelle) [inline README.md:62]: Oooo, nice color!

File: index.js [hunk 2]
```diff
@@ -4,15 +4,37 @@ console.log("Here's your header:", header)
...
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+const profile = document.querySelector('div#profile')
+const travelerImg = profile.querySelector('img')
 
+travelerImg.src = traveler.photo
+travelerImg.alt = traveler.name
+
+profile.querySelector('h2').innerHTML = traveler.name
+profile.querySelector('em').innerHTML = traveler.nickname
```
Comment (BrendaMichelle) [inline index.js:19]: Nice organization and good thinking scoping your querySelector search to the profile (:

Comment (BrendaMichelle) [reply index.js:19]: Always use `textContent` over `innerHTML` where possible, especially if you know you're setting plain text. See here to read about the security implications of `innerHTML`: https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML#security_considerations

```suggestion
console.log('This is what the traveler object looks like: ', traveler)
const profile = document.querySelector('div#profile')
const travelerImg = profile.querySelector('img')

travelerImg.src = traveler.photo
travelerImg.alt = traveler.name

profile.querySelector('h2').textContent = traveler.name
profile.querySelector('em').textContent = traveler.nickname
```

File: index.js [hunk 3]
```diff
@@ -4,15 +4,37 @@ console.log("Here's your header:", header)
...
+        const unorderedList = document.querySelector('ul#animals')
+        const animalSighting = 
+            `<li data-id="${sighting.id}">
+                <p>${sighting.description}</p>
+                <img src="${sighting.photo}" alt="${sighting.species}"/>
+                <a href="${sighting.link}" target="_blank">Here's a video about the ${sighting.species} species!</a>
+            </li>`
+        unorderedList.innerHTML += animalSighting
+    }
+)
```
Comment (BrendaMichelle) [inline index.js:35]: Nice! Try the "combo" or "artisanal" approach that we learned in lecture over the `innerHTML +=` approach to make the code more efficient and safer (especially, with the "artisanal" approach)

File: index.js [hunk 4]
```diff
@@ -4,15 +4,37 @@ console.log("Here's your header:", header)
...
+
 
 /***** Deliverable 5 *****/
+const wrongTraveler = document.querySelector("[data-id='3']")
+wrongTraveler.remove()
```
Comment (BrendaMichelle) [inline index.js:40]: good!

**Review by BrendaMichelle — COMMENTED** [review]
Great work!

File: index.js (hunk 3 above)
Comment (anhcdang) [reply index.js:35]: Thank you, Michelle! What changes should I make to achieve the combo approach? I was under the impression that this is the combo approach.

**Pull request closed without merging**